test/distributed/_composable/test_fully_shard.py

```diff
@@ -251,7 +250,5 @@ def test_training(self):
             self.assertEqual(losses[0], losses[1])
 
 
-instantiate_parametrized_tests(TestFSDPInitialization)
```

Comment:
I am wondering: Does this cause overhead or something? Just curious what is the motivation for removing this.

Reply:
The overhead should be negligible. But it looks like there is no `@parametrize` in the `TestFSDPInitialization`. Or is it intentional to keep this, just in case people add `@parametrize` in the future?

Reply:
Yes, I was originally thinking to keep it in case we add `@parametrize` in the future, but removing it is totally fine.